Event.js

```diff
@@ -1,22 +1,57 @@
-//creare un input selezione tramire CreateElement("")collegandolo all'html
-// al click-->(addEventListener())parte la fetch
-//dove l'utente puo scegliere gli invitati, tramite fetch
-//cioè tutti quelli che si sono iscritti all'app
-const response = [
-  { id: 1, name: "Ani" },
-  { id: 2, name: "Eni" },
-  { id: 3, name: "Fulvio" },
-];
-
-const select = document.querySelector("#guest");
-
-for (const i of response) {
-  const options = document.createElement("option");
-  options.text = i.name;
-  select.options.add(options);
+const form_event = document.querySelector("#form_event");
+//creare una fetch per richiamare i dati dell'utente in corso
+
+// prendiamo il token identificativo per farci riconoscere dal server (il getItem vuole solo la chiave)
+const current_token = localStorage.getItem("access_token");
+
+async function getMe() {
+  const me_response = await fetch("https://iopasso.kopiro.me/me", {
+    method: "GET",
+    headers: {
+      authorization: "Bearer " + current_token,
+    },
+  });
+  const me_json = await me_response.json();
+  const user_online = me_json.name;
+  console.log(user_online);
+  return user_online;
+}
+
+getMe();
+
+//creo asyc funnction per prendere con la fetch in get la lista di tutti gli utenti
+async function listGuests() {
+  const guests_response = await fetch("https://iopasso.kopiro.me/guests", {
+    method: "GET",
+    headers: {
+      authorization: "Bearer " + current_token,
+    },
+  });
+  // trasformiamo la lista in json
+  const guest_json = await guests_response.json();
+
+  //prendiamo l'elemento select dall´html
+  const select = document.querySelector("#guest");
+  //cicliamo la risposta della fetch in get per prenderci solo il nome dell'utente
+  for (const i of guest_json) {
```

Comment:
Never call an item value in a for in a generic way like `i`, please rename it to something specific like `guest`.

It's ok to call an item `i` if it's the count of the `for (let i=0; i < len; i++)`